powersimdata/design/generation/tests/test_cost_curves.py

```diff
@@ -174,35 +132,14 @@
         1200,
         1900,
     ],
-    "interconnect": [
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-        "D",
-    ],
+    "interconnect": ["Western"] * 20,
 }
 
 grid_attrs = {"plant": mock_plant, "gencost_before": mock_gencost}
 
 grid = MockGrid(grid_attrs)
-grid.interconnect = "D"
-grid.zone2id = {"A": 0, "B": 1, "C": 2}
+grid.interconnect = "Western"
+grid.zone2id = {"Utah": 210, "Colorado": 212, "Washington": 201}
```

Comment:
Are we still able to use these sort of 'dummy' grids for testing?

Reply:
I would say yes. `interconnect`, `zone2id` and `id2zone` are still attributes of the `Grid` class. The function in the `powersimdata.design.generation.cost_curves` module take a `Grid` object as a parameter so it is all good. The user will instantiate the `Grid` class with the grid model of his/her/their choice (via the `source` paramater. Default is set to `usa_tamu`) outside of these functions.

Reply:
So even without the changes from zones named `"A", "B", "C"`, to `"Utah", "Colorado", "Washington"` the code should still work (assuming that we give the `MockGrid` the appropriate info)?

Reply:
No it won't work because the functions in `powersimdata.design.generation.cost_curves` use the `area_to_loadzone` function that is now independent of a `Grid` object. The `area_to_loadzone` function uses the dictionaries in `powersimdata.network.GRID_MODEL.constants.zones` were real loadzones are defined

Reply:
So if we want to use dummy grids for any functions that call `area_to_loadzone`, then we would need to define parameters for those dummy grids in `powersimdata.network.DUMMY.constants.zones`?

Reply:
I know it is not optimal for testing. We talked about creating a `MockConstants` in `powersimdata.tests` with @BainanXia. It seems dumb or perhaps necessary if we don't want to instantiate a `Grid` class each time we want to have a mapping from a kind of zone to another one.

Reply:
This will be tackled in a different PR